README.md

```diff
@@ -4,5 +4,5 @@ KoC scripts for customizing distributions
 ## How to test the script
 
 ```
-curl -sSL https://raw.githubusercontent.com/kidsoncomputers/kocifier/master/kocify.sh | bash
+curl -sSL https://raw.githubusercontent.com/roninsphere/kocifier/mexico2019/kocify.sh | bash
```

Comment:
Does this need to switch back to `../kidsoncomputers/kocifier/..`, to go with the changes you've made in this PR to `kocify.sh` ?

Reply:
Yes, the packages.txt and the script kocify.sh are on my fork; the urls will need to change back to kidsoncomputers when merged.

Reply:
Yes, the URL's switch back when merged.